test/extended/util.go

```diff
@@ -0,0 +1,23 @@
+package extended
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/GoogleCloudPlatform/kubernetes/test/e2e"
+)
+
+var testContext = e2e.TestContextType{}
+
+func kubeConfigPath() string {
+	return os.Getenv("KUBECONFIG")
+}
+
+func writeTempJSON(path, content string) error {
```

Comment:
Seems weird to have a `WriteTempJSON` that doesn't have to write to a temporary file.  Why not have it choose a temp file and return the path it wrote?